cross-region-replication-mm2-converter/src/main/java/com/amazonaws/services/crossregion/schemaregistry/kafkaconnect/CrossRegionReplicationMM2Converter.java

```diff
@@ -0,0 +1,170 @@
+package com.amazonaws.services.crossregion.schemaregistry.kafkaconnect;
+
+import com.amazonaws.services.schemaregistry.common.AWSSerializerInput;
+import com.amazonaws.services.schemaregistry.common.Schema;
+import com.amazonaws.services.schemaregistry.common.configs.GlueSchemaRegistryConfiguration;
+import com.amazonaws.services.schemaregistry.deserializers.GlueSchemaRegistryDeserializationFacade;
+import com.amazonaws.services.schemaregistry.deserializers.GlueSchemaRegistryDeserializerImpl;
+import com.amazonaws.services.schemaregistry.exception.AWSSchemaRegistryException;
+import com.amazonaws.services.schemaregistry.serializers.GlueSchemaRegistrySerializationFacade;
+import com.amazonaws.services.schemaregistry.serializers.GlueSchemaRegistrySerializerImpl;
+import com.amazonaws.services.schemaregistry.utils.AWSSchemaRegistryConstants;
+
+import kotlinx.serialization.SerializationException;
+import lombok.Data;
+import lombok.Getter;
+
+import org.apache.kafka.connect.data.*;
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.storage.Converter;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Data
+public class CrossRegionReplicationMM2Converter implements Converter {
```

Comment:
Good to add class level documentation on how this works. 

We can remove references to MM2 as I don't see anything specific to MM2 here. This can be used with non-MM2 like connectors that need cross-region schemas. Please rename the class and update documentation accordingly.